utilities/src/main/java/io/syndesis/qe/utils/CalendarUtils.java

```diff
@@ -0,0 +1,26 @@
+package io.syndesis.qe.utils;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.Calendar;
+
+/**
+ * This class is only for saving time before and after request. After that, the time in the any step can be compare
+ * whether it is between these two date.
+ */
+@Slf4j
+@Component
```

Comment:
You could use @Data annotation here. It's just syntactic sugar from lombok - no need to use the Getter and Setter annotations, nothing to do with the functionality.